fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -106,9 +106,9 @@ private void exportBinary(final URI uri)
             writeResponse(response, file);
         }
     }
-    private void exportContainer(final URI uri)
+    private void exportDescription(final URI uri, final URI... describes)
             throws FcrepoOperationFailedException, IOException {
-        final File file = fileForContainer(uri);
+        final File file = fileForContainer(describes.length > 0 ? describes[0] : uri);
```

Comment:
I definitely think we should serialize all of the descriptions.  Assuming we're talking only about current Fedora, then we're already doing that.  But since a future Fedora or another LDP implementation could have multiple descriptions, then handling multiples seems like a good idea to me.

AFAIK, the only platform that has issues with colons is MacOSX.  And filenames with colons work fine for me using the default filesystem type (HFS+ journaled).  We can always replace the colons with underscores or something to be sure it won't cause problems.

I think having the filenames based on the description URI also has the benefit of replacing the file when it's updated.  If we use numbered suffixes, then we wouldn't necessarily know which description went with which file and they might not get updated correctly because of that.